docs/data-access/data-download.md

```diff
@@ -17,38 +19,96 @@ Platform users can search for a file set of interest using the File Repository.
+
+### Distributions
+<!---  Tabs start here -->
+
+<Tabs
+groupId="operating-systems"
+defaultValue="Client"
+values={[
+{ label: 'Client', value: 'Client', },
+{ label: 'Client with ENV variables', value: 'Client-ENV', },
+{ label: 'Docker', value: 'Docker', },
+]
+}>
+<TabItem value="Docker">
 
 Pull the latest version of the score-client Docker distribution:
 
-```
+```shell
 > docker pull overture/score
 ```
+Once pulled, you can open a shell in the container by executing:
 
-Use the `docker run` command with the correct variables specified. You will need to define:
+```shell
+> docker run -it overture/score
+> score-client
+``` 
+
+Update the docker configuration with your user values, including:
 
+- **METADATA_URL**: the file metadata Song server URL
+- **STORAGE_URL**: the object storage Score server URL
 - **ACCESSTOKEN**: your personal [API Token](/docs/data-access/user-profile-and-api-token)
+
+
+```shell 
+> docker run -it -e "METADATA_URL=https://song.rdpc.cancercollaboratory.org" -e "STORAGE_URL=https://score..rdpc.cancercollaboratory.org" -e "ACCESSTOKEN=92038829-338c-4aa2-92fc2-a3c241f63ff0" overture/score 
+```
+
+There is no entry point or command defined for the image. The software is located at score-client which is also the working directory of the container. 
+
+
+</TabItem>
+<TabItem value="Client-ENV">
+
+Download the **[latest version of the score-client](https://artifacts.oicr.on.ca/artifactory/dcc-release/bio/overture/score-client/[RELEASE]/score-client-[RELEASE]-dist.tar.gz)**. Once you have unzipped the tarball, change directories into the unzipped folder:
+
+```shell
+> wget -O score-client.tar.gz https://artifacts.oicr.on.ca/artifactory/dcc-release/bio/overture/score-client/[RELEASE]/score-client-[RELEASE]-dist.tar.gz
+
+> tar xvzf score-client.tar.gz
+
+## Note: Once unzipped, the final directory will be suffixed with the latest release number.
+> cd score-client-<latest-release-number>
+```
+
+You can define required inputs as ENV variable, stored on system or defined with each client operation:  
+
 - **METADATA_URL**: the file metadata Song server URL
 - **STORAGE_URL**: the object storage Score server URL
-- the absolute directory path where your file manifest is located
-- an output directory path
+- **ACCESSTOKEN**: your personal [API Token](/docs/data-access/user-profile-and-api-token)
+
+
+For example to download files with a manifest:
 
 ```shell
-> docker run --rm -it -e "METADATA_URL=https://song.rdpc.cancercollaboratory.org" -e "STORAGE_URL=https://score..rdpc.cancercollaboratory.org" -e "ACCESSTOKEN=92038829-338c-4aa2-92fc2-a3c241f63ff0" -v "C:\Users\username\Desktop\directory-path\" overture/score:latest score-client download --manifest /directory-path/score-manifest.20200520.tsv --output-dir C:\Users\username\Desktop\download\
+> METADATA_URL=https://api.platform.icgc-argo.org/storage-api STORAGE_URL=https://api.platform.icgc-argo.org/storage-api score-client download --manifest manifest1.txt
```

Comment:
If this distribution example is illustrating how to use score client after `cd score-client-<latest-release-number>` I think it should be `...storage-api score-client download...` -> `...storage-api bin/score-client download...`. If used as is it will complain that `score-client` is not found. Not sure what kind of users would read this doc so maybe this is a non-issue.

Reply:
good catch thank you